nrfx/hal/nrf_nvmc.h

```diff
@@ -133,6 +134,44 @@ NRF_STATIC_INLINE void nrf_nvmc_nonsecure_mode_set(NRF_NVMC_Type *    p_reg,
                                                    nrf_nvmc_ns_mode_t mode);
 #endif
 
+/**
+ * @brief Function for writing a word into the flash
```

Comment:
Let's use same scheme as in `nrfx_nvmc.h` :
` * @brief Function for writing a 32-bit word to flash.`